scheduler/scheduler-server/src/test/java/functionaltests/db/schedulerdb/TestLoadJobsPagination.java

```diff
@@ -125,7 +125,9 @@ public void testSorting() throws Exception {
                                  null,
                                  null,
                                  null,
-                                 sortParameters(new SortParameter<>(JobSortParameter.ID, SortOrder.ASC)))
+                                 sortParameters(new SortParameter<>(JobSortParameter.ID, SortOrder.ASC)),
```

Comment:
In this class I think you should add a test with the new parameters.

The class has a helper method which allows to add a submitted job with a submittedTime parameter